mantidproject/mantid
# Load spectra mask file applying it to the detector map.

This is part of wider ticket #16808 to allow ISIS direct inelastic group to use xml-based detector maps.

**To test:**
The changes in functionality are probably best described by changed LoadMask algorithm description and its doctest so the part of the testing is to understand if the documentation is clear



Check if new functionality works without side effects and as intended. Check if the description of the new functionality is clear enough. 

Fixes #16848.

The changes are not particularly useful without #16808 so all release notes will be there. 

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
